conf.yaml

```diff
@@ -1363,14 +1363,34 @@ contents:
                         repo:   apm-agent-rum-js
                         path:   docs
           -
-            title:      Infrastructure Monitoring Guide
-            prefix:     en/infrastructure/guide
-            current:    7.4
-            branches:   [ master, 7.x, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5 ]
-            index:      docs/en/infraops/index.asciidoc
+            title:      Logs Monitoring Guide
+            prefix:     en/logs/guide
+            current:    master
+            branches:   [ master ]
```

Comment:
Should we also build `7.5` here? Then update that to `current` when `7.5` releases?

Reply:
Comment as below

Reply:
👍